crates/biome_js_analyze/tests/specs/nursery/useVueMultiWordComponentNames/invalid-has-name.vue.snap

```diff
@@ -0,0 +1,29 @@
+---
+source: crates/biome_js_analyze/tests/spec_tests.rs
+expression: invalid-has-name.vue
+---
+# Input
+```ts
+export default {
+	name: "invalid",
+}
+
+```
+
+# Diagnostics
+```
+invalid-has-name.vue:1:2 lint/nursery/useVueMultiWordComponentNames ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  × This Component's name "invalid" only contains one word.
+  
+  > 1 │ export default {
+      │  ^^^^^^^
```

Comment:
It seems that the  range is a little off. That's weird

Reply:
Yes, but it's unrelated to this PR. See also: #4593